rhc-ose-ansible/roles/openstack-create/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+env_id: "testenv"
+openstack_master_security_group: "ose3-master"
+openstack_node_security_group: "ose3-node"
+openstack_flavor_id: "d5fbd66f-2069-42d6-9bde-254bac4617b1"
```

Comment:
I'll add functionality to perform the lookup of the id and append the branch and will notify when changes have been pushed